pset-functions.js

```diff
@@ -114,6 +188,13 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+function isPalindrome(word){
+  return word === word.split("").reverse().join("")
```

Comment:
Really awesome use of method chaining and putting it all into a function so you can continue to re-use the code!